src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
+
+function getSalary(element) {
+  const salary = element.dataset.salary;
+
+  if (salary) {
+    let cleanedSalary = salary.slice(1);
+
+    cleanedSalary = cleanedSalary.split(',').join('');
+
+    return Number(cleanedSalary);
+  }
+
+  return 0;
+}
+
+function sortList(list) {
+  const listItem = [...list];
+
+  return listItem.sort((a, b) => getSalary(b) - getSalary(a));
+}
+
+listEmployees.innerHTML = '';
+
+const sortedList = sortList(employees);
```

Comment:
The function `sortList` is modifying the original `employees` array. It's a good practice to avoid mutating the original data. Consider creating a copy of the array inside `sortList` function before sorting it.